lib/src/test/kotlin/starknet/account/StandardAccountTest.kt

```diff
@@ -1297,4 +1312,73 @@ class StandardAccountTest {
             assertEquals(1, messages[1].order)
         }
     }
+
+    @Nested
+    inner class OutsideExecutionTest {
+        private val randomAddress = Felt.fromHex("0x655b2696f187daec1952ef59befbd875add757748d6c844fb273439cd0d82f9")
```

Comment:
If this is a purely random (not actual one) address, then let's use a dummy value (e.g. `0x123`).